PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/keras-team/keras-contrib/blob/master/CONTRIBUTING.md
+-->
+
+### Summary
+
+### Related Issues
+
+### PR Overview
+
+- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
+- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
+- [ ] This PR is backwards compatible [y/n]
+- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)
```

Comment:
For the PR template, I think we're going to go with the new design which you can find here: https://github.com/keras-team/keras/pull/12169

Reply:
@gabrieldemarmiesse Some sections don't appear when you open the pull request (those with shape <!-- XXX -->). Is it a wanted behaviour or should we fix this?

Reply:
All the `<!-- XXX -->` are comments on Github and do not show up when opening a PR, but there are in the pull request template file. What should we do with them?